### PR TITLE
create fixture enum for acceptance tests

### PR DESCRIPTION
**Pull request by danyf90**
### Short description 📝

Improves types safety and discoverability a bit. It would be nice to generate the enum

### Contributor checklist ✅

- [x] The code has been linted using run `make workspace/lint-fix`
- [x] The change is tested via unit testing or acceptance testing, or both
- [x] The title of the PR is formulated in a way that is usable as a changelog entry
- [x] In case the PR introduces changes that affect users, the documentation has been updated

### Reviewer checklist ✅

- [x] The code architecture and patterns are consistent with the rest of the codebase
- [x] Reviewer has checked that, if needed, the PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, and the title is usable as a changelog entry

### CODE REVIEW

File: Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift [hunk 1]
```diff
@@ -0,0 +1,188 @@
+//
+//  TuistAcceptanceFixtures.swift
+//  TuistAcceptanceTesting
+//
+//  Created by Daniele Formichelli on 20/12/23.
+//
+
+import Foundation
```
Comment (pepicrft) [inline Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift:8]: Let's remove the file headers

File: Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+public enum TuistAcceptanceFixtures {
+    case appWithBuildRules
```
Comment (pepicrft) [inline Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift:11]: In a follow-up we can document this code and use Swift DocC to generate a reference that includes what every fixture is about :)

**Review by pepicrft — APPROVED** [review]

File: Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift (hunk 2 above)
Comment (danyf90) [reply Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift:11]: 👍

**Review by fortmarek — APPROVED** [review]
Thanks, this is definitely an improvement.

**Pull request merged**